app/models/worldwide_office.rb

```diff
@@ -1,24 +1,22 @@
 class WorldwideOffice < ApplicationRecord
   has_one :contact, as: :contactable, dependent: :destroy
-  belongs_to :worldwide_organisation
   belongs_to :edition
   has_many :worldwide_office_worldwide_services, dependent: :destroy, inverse_of: :worldwide_office
   has_many :services, through: :worldwide_office_worldwide_services, source: :worldwide_service
-  validates :worldwide_organisation, :contact, :worldwide_office_type_id, presence: true
-  validate :worldwide_organisation_or_edition
+  validates :contact, :edition, :worldwide_office_type_id, presence: true
 
   after_commit :republish_embassies_index_page_to_publishing_api
 
   accepts_nested_attributes_for :contact
 
-  include PublishesToPublishingApi
```

Comment:
How is this now published if it's not itself an edition? Does it get published via the editionable WWO?

Reply:
It's now published by [`PublishingApiAssociatedDocuments`](https://github.com/alphagov/whitehall/blob/8eceacc2024becb728f2e14a3677f655d3e04085/app/services/service_listeners/publishing_api_associated_documents.rb) and we define the offices as associated documents [here](https://github.com/alphagov/whitehall/blob/8eceacc2024becb728f2e14a3677f655d3e04085/app/models/editionable_worldwide_organisation.rb#L209).

Reply:
Interesting!